backend/src/bundles/achievements/achievement.model.ts

```diff
@@ -0,0 +1,19 @@
+import {
+    AbstractModel,
+    DatabaseTableName,
+} from '~/common/database/database.js';
+
+import { type ActivityType, type Metric, type ValueOf } from './enums/enums.js';
+
+class AchievementModel extends AbstractModel {
```

Comment:
let's add relations mappings for this model

Reply:
I also added relations to ...backend\src\bundles\users\user.model.ts